assets/scss/components/stencil/applePay/_applePay.scss

```diff
@@ -0,0 +1,54 @@
+$apple-pay-black: black;
+$apple-pay-white: white;
+
+.apple-pay-checkout-button {
+    background-position: 50% 50%;
+    background-repeat: no-repeat;
+    background-size: 100% 60%;
+    border-radius: 4px;
+    box-sizing: border-box;
+    cursor: pointer;
+    display: none;
+    max-height: 64px;
+    min-height: 32px;
+    min-width: 134px;
+    padding: 20px;
+    @if stencilString("button--apple-pay") == "white" {
```

Comment:
Never mind, just realized this wasn't also a class. Would be good to follow convention here too though, so 🍹